docker-compose.yml

```diff
@@ -24,10 +24,6 @@ services:
       - ALL
     volumes:
       - ./config.json:/app/config.json:ro
-      - ./config_devnet.json:/app/config_devnet.json:ro
-      - ./config_comnet.json:/app/config_comnet.json:ro
       - ./peering.json:/app/peering.json
       - ./profiles.json:/app/profiles.json:ro
-      - ./mainnetdb:/app/mainnetdb
-      - ./p2pstore:/app/p2pstore
-      - ./snapshots/mainnet:/app/snapshots/mainnet
+      - ./stardust_testnet:/app/stardust_testnet
```

Comment:
I actually don't like that the network name is hardcoded inside the docker container

Reply:
I would leave it as it is for now for the alpha tests.